text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndExport.scala

```diff
@@ -64,11 +72,25 @@ case class SerializedExporter(filename: String) extends Exporter {
   override def close(): Unit = ()
 }
 
+case class JSONExporter(filename: String) extends Exporter {
+  override def export(mentions: Seq[Mention]): Unit = {
+    Serializer.save[SerializedMentions](new SerializedMentions(mentions), filename + ".json");
```

Comment:
Let's delete this line because it is unnecessary for json serialization. It is actually using java serialization to write the mentions to the `.json` file, which is later overwritten with the actual json content (line 82).